test/appium/tests/critical/test_wallet.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+from _pytest.outcomes import Failed
+from selenium.common import TimeoutException
+
+from base_test_case import MultipleSharedDeviceTestCase, create_shared_drivers
+from support.api.network_api import NetworkApi
+from tests import marks, run_in_parallel
+from users import transaction_recipients
+from views.sign_in_view import SignInView
+
+
+@pytest.mark.xdist_group(name="new_four_2")
+@marks.new_ui_critical
+class TestWalletMultipleDevice(MultipleSharedDeviceTestCase):
+
+    def prepare_devices(self):
+        self.network_api = NetworkApi()
+        self.drivers, self.loop = create_shared_drivers(2)
+        self.sign_in_1, self.sign_in_2 = SignInView(self.drivers[0]), SignInView(self.drivers[1])
+        self.sender, self.receiver = transaction_recipients['J'], transaction_recipients['K']
+        self.sender_username, self.receiver_username = 'sender', 'receiver'
+        self.loop.run_until_complete(
+            run_in_parallel(((self.sign_in_1.recover_access, {'passphrase': self.sender['passphrase'],
+                                                              'username': self.sender_username}),
+                             (self.sign_in_2.recover_access, {'passphrase': self.receiver['passphrase'],
+                                                              'username': self.receiver_username}))))
+        self.home_1, self.home_2 = self.sign_in_1.get_home_view(), self.sign_in_2.get_home_view()
+        self.wallet_1 = self.home_1.wallet_tab.click()
+        self.wallet_2 = self.home_2.wallet_tab.click()
+
+    # ToDo: Add verification of Activity tabs when the feature is ready in the next 2 tests:
+
+    def _get_balances_before_tx(self):
```

Comment:
are you sure it is worth having it here, not in a separate file?

I believe it might be used in subsequent tests too?

Reply:
I'd rather leave as it is for now as it's needed only here. If we'll use in other files we can move it later